iac-extensions/kubernetes/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+plugins {
+    id("org.sonarsource.iac.code-style-convention")
+    id("org.sonarsource.iac.java-conventions")
+}
+
+description = "SonarSource IaC Analyzer :: Extensions :: Kubernetes"
+
+dependencies {
+    api(project(":iac-common"))
+    implementation(project(":sonar-helm-for-iac"))
+    implementation(libs.google.protobuf)
+
+    testImplementation(libs.junit.jupiter)
+    testImplementation(libs.assertj.core)
+    testImplementation(libs.mockito.inline)
+    testImplementation(libs.sonar.plugin.api.test.fixtures)
+    testImplementation(testFixtures(project(":iac-common")))
+}
+
+tasks.register<Exec>("compileProtobufJava") {
+    description = "Compile the protobuf for Java."
+    group = "build"
+
+    inputs.files("${project.projectDir}/../../sonar-helm-for-iac/template-evaluation.proto")
+    outputs.dir("build/generated-sources")
+    outputs.cacheIf { true }
+
+    commandLine(
+        "protoc",
+        "-I=${project.projectDir}/../../sonar-helm-for-iac/",
+        "-I=${System.getProperty("user.home")}/go/protobuf/include",
+        "--java_out=${project.projectDir}/build/generated-sources",
+        "${project.projectDir}/../../sonar-helm-for-iac/template-evaluation.proto"
+    )
+}
+
+tasks.named("compileJava") {
+    dependsOn("compileProtobufJava")
+}
+
+tasks.named("sourcesJar") {
+    dependsOn("compileProtobufJava")
+}
+
+tasks.processTestResources {
+    dependsOn(":sonar-helm-for-iac:compileGoCode")
+}
+
+sourceSets {
+    main {
+        java {
+            srcDir("build/generated-sources")
+        }
+    }
+    test {
+        resources {
+            srcDir("../../sonar-helm-for-iac/build/executable")
```

Comment:
This can be done by adding a custom configuration to `sonar-helm-for-iac` and attaching the artifact to it, but it can also be changed later